extension/src/postcss.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = () => ({
+  plugins: {
+    "@tailwindcss/postcss": {},
+    autoprefixer: {},
```

Comment:
looks like using postcss to implement tailwind's animation lib is the cleanest way to integrate tailwind into our build system